linode/manager
# fix: API Tokens Table Style Regression

## Description 📝

Fixes a small style regression caused by design tokens 🎨 on the API tokens page

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-09-11 at 11 02 52 AM](https://github.com/user-attachments/assets/1f5e7e22-977b-4c2f-8164-36a2f24dfb9d) | ![Screenshot 2024-09-11 at 11 02 58 AM](https://github.com/user-attachments/assets/05f07c73-50d5-4732-bfa3-cd499d870fe8) |

## How to test 🧪

- Go to http://localhost:3000/profile/tokens
- Check styles on this page 👀 
  - Test in light 🌞  and dark 🌑  mode
  - Test on small screen sizes 📱 
  
## As an Author I have considered 🤔
